### PR TITLE
Add docs for cattle detector in prebuilt.md file

### PR DESCRIPTION
**Pull request by Mu-Magdy**
Adding cattle detector to docs

### CODE REVIEW

File: docs/prebuilt.md [hunk 1]
```diff
@@ -55,6 +55,12 @@ Table S1 Confusion matrix for the Alive/Dead model in Weinstein et al. 2023
 
 Citation: Weinstein, Ben G., et al. "Capturing long‐tailed individual tree diversity using an airborne imaging and a multi‐temporal hierarchical model." Remote Sensing in Ecology and Conservation 9.5 (2023): 656-670.
 
+## Livestock Detectors
+
+The goal of this project is to detect livestock using drone and UAV imagery. To ensure that the model generalizes well across different points of view and camera types, various datasets will be used. This approach aims to enhance the model's ability to perform accurately under diverse conditions and perspectives.
```
Comment (bw4sz) [inline docs/prebuilt.md:60]: Don't use future tense, once its up on the website it should be 'available', so it reads. 

# Livestock Detector

This model has a single label 'livestock' trained on drone imagery of cows, sheep and other large mammals in agricultural settings. The model was trained on data from [insert countries and other metadata about landscapes].

**Comment by bw4sz** [comment]
Looks good, I added a text edit. @ethanwhite this is part of that conversation about docs building from main versus release. Should we wait until https://github.com/weecology/DeepForest/issues/746 is merged before we merge this? I think so, users might see the model and the model isn't ready until it has been 

- [ ] Uploaded to hugging face @henrykironde 
- [ ] #317 on hugging face is complete
- [ ] A example of how to run is complete. @Mu-Magdy we will add this example right below each model once items 1 and 2 are complete.

**Comment by ethanwhite** [comment]
I'm fine with waiting, but I also think that @henrykironde should have that sorted out soon enough that we don't need to wait on it. @henrykironde if you're having issues getting #746 to default to `stable` let me know on Slack and I can take a look.

**Review by bw4sz — APPROVED** [review]
I am happy with the docs, we will need to add an example and a name structure once @henrykironde get #317 closed.

**Comment by bw4sz** [comment]
I'm merging this. @Mu-Magdy your next step is to work with @henrykironde to get the model uploaded to hugging face and add an example on how to call it in the new interface.

https://huggingface.co/weecology
https://github.com/weecology/DeepForest/blob/5b2b5e2ae5f5aa523ce655572cdb5980de151d53/deepforest/utilities.py#L59

**Pull request merged**